packages/core/src/components/Onboarding/index.tsx

```diff
@@ -19,6 +19,7 @@ import { PickPrimary } from './pages/PickPrimary'
 import { Referral } from './pages/Referral'
 import { ReviewPrimary } from './pages/ReviewPrimary'
 import { Welcome } from './pages/Welcome'
+var namer = require('color-namer')
```

Comment:
Does this library not work with module imports ( the other imports we have - `import X from Y`) instead of using the commonjs ( `var x = require('y')` ) ones? In any case I also think it should be `const namer = ` instead of `var namer = ` ( The official documentation says var, but its 4 years old now. `const` is definitely better here )